Gemfile

```diff
@@ -57,7 +57,7 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'devise'
 
-# gem 'omniauth-rails_csrf_protection'
+gem 'omniauth-rails_csrf_protection'
```

Comment:
Do google authentication without this gem

Reply:
sir iss gem ke bina ye work nhi kr rha

Reply:
it shows me authenticity error